src/utils/aria/aria.ts

```diff
@@ -17,6 +17,7 @@ type ariaProp = { [key in ariaPropName]: string } | {};
 
 /**
  * Returns object with 'aria-' property which can be spread to props.
```

Comment:
I'd maybe clarify the fact that using this util is a good way to prevent empty aria values when the conditions are not met. So I'd add something along the lines of: "prevents generation of empty 'aria-' properties when no valid values are given."